### ISSUE TITLE
WAS runtime cannot be found/loaded by apps on Windows Server 2019 LTSC

### ISSUE THREAD

**Issue by riverar** [microsoft/WindowsAppSDK#1121]
**Describe the bug**
After installing the WAS runtime on Windows Server 2019 LTSC, apps attempting to bootstrap cannot locate the WAS runtime.

Are W.A.M.AppExtensions supported on LTSC? This doesn't work either (PowerShell):
```powershell
[Windows.ApplicationModel.AppExtensions.AppExtensionCatalog,Windows.ApplicationModel,ContentType=WindowsRuntime]::Open(
"com.microsoft.reunion.ddlm-0.8-x64-p")
```

**Steps to reproduce (C++)**
TBD

**Steps to reproduce (Rust)**
1. Enable sideloading of packages via Settings
2. Deploy runtime .\ProjectReunion-0.8Preview-Install-x64.exe
3. `git clone https://github.com/microsoft/windows-samples-rs/`
4. `cd windows-samples-rs/windows_app_sdk/skeleton`
5. `cargo run`

**Steps to reproduce (Pre-compiled, x64)**

1. Enable sideloading of packages via Settings
2. Deploy runtime .\ProjectReunion-0.8Preview-Install-x64.exe
3. Download [precompiled_skeleton.zip](https://github.com/microsoft/WindowsAppSDK/files/6876148/precompiled_skeleton.zip)
4. Run `skeleton.exe`
5. Observe (self-imposed) error due to internal DDLM 0x80070490 "Element not found."

ProjectReunion-0.8Preview-Install-x64.exe
Windows Server 2019 17763.1

**Comment by Zymlex** [comment]
Same problem.

**Comment by DrusTheAxe** [comment]
Manifested `windows.appExtension` is in the uap3 namespace (i.e. a long time ago) and the `AppExtension` API is likewise supported for a long time so it's probably a server-specific issue.

>After installing the WAS runtime on Windows Server 2019 LTSC

Did the installer successfully install the packages? You can verify from a command line
```
$(get-appxpackage *reunion*).packagefullname
```
and you should see the 2x Frameworks (x86+x64), 1x Main (x64) and 2x DDLM packages (x86+x64).

Do you have the Desktop Experience(?) enabled?

**Comment by Zymlex** [comment]
```
Microsoft.ProjectReunion.0.8-preview_8000.146.628.0_x86__8wekyb3d8bbwe
Microsoft.ProjectReunion.0.8-preview_8000.146.628.0_x64__8wekyb3d8bbwe
Microsoft.ProjectReunion.Main.0.8-preview_8000.146.628.0_x64__8wekyb3d8bbwe
Microsoft.ProjectReunion.DDLM.8000.146.628.0-x8-p_8000.146.628.0_x86__8wekyb3d8bbwe
Microsoft.ProjectReunion.DDLM.8000.146.628.0-x6-p_8000.146.628.0_x64__8wekyb3d8bbwe
```

> Do you have the Desktop Experience(?) enabled?

Yes

**Comment by riverar** [comment]
Yep, packages are installed.
```
Microsoft.ProjectReunion.0.8-preview_8000.146.628.0_x86__8wekyb3d8bbwe
Microsoft.ProjectReunion.0.8-preview_8000.146.628.0_x64__8wekyb3d8bbwe
Microsoft.ProjectReunion.Main.0.8-preview_8000.146.628.0_x64__8wekyb3d8bbwe
Microsoft.ProjectReunion.DDLM.8000.146.628.0-x8-p_8000.146.628.0_x86__8wekyb3d8bbwe
Microsoft.ProjectReunion.DDLM.8000.146.628.0-x6-p_8000.146.628.0_x64__8wekyb3d8bbwe
```

WS19 LTSC w/ desktop experience.

**Comment by DrusTheAxe** [comment]
Does this issue repro with 1.0-preview1?

**Comment by riverar** [comment]
@DrusTheAxe Will get you an answer shortly.

**Comment by riverar** [comment]
@DrusTheAxe Same issue with `1.0-preview1`. I'm not entirely convinced AppExtensions work at all on LTSC.

PowerShell exercising W.A.M.AppExtensions 
```powershell
[Windows.ApplicationModel.AppExtensions.AppExtensionCatalog,Windows.ApplicationModel,ContentType=WindowsRuntime]::Open("microsoft.winappruntime.ddlm-0.258.2116.0-x6-p1")
```

Steps to reproduce (Rust)
1. Enable sideloading of packages via Settings
2. Deploy preview1 runtime for x86/x64
3. `git clone https://github.com/riverar/was-repros`
4. `cd was_issue_1121`
5. `cargo run --release`

**Comment by riverar** [comment]
@jvintzel Hey John, do you know if AppExtensions has known quirks on LTSC or is supported?

**Comment by jvintzel** [comment]
App Extensions are supported on 1809 LTSC and later.  I am unaware of any quirks for them.

**Comment by riverar** [comment]
Err @DrusTheAxe is this expected to work for exp/prev releases?

https://github.com/microsoft/WindowsAppSDK/blob/5661c2dedcb018612ee105c87cd5a629056df5a7/dev/DynamicDependency/API/MddLifetimeManagement.cpp#L121-L126

**Comment by DrusTheAxe** [comment]
> I'm seeing the same as [#1121 (comment)](https://github.com/microsoft/WindowsAppSDK/issues/1121#issuecomment-934640347) on desktop Windows 1909.

I suspect this is the same root problem. Software archeology in progress (i.e. I'm digging :P) to verify Windows Server 2019 LTSC is based on 19H1 then yes, this should be addressed by #1681

**Comment by DrusTheAxe** [comment]
> > I'm seeing the same as [#1121 (comment)](https://github.com/microsoft/WindowsAppSDK/issues/1121#issuecomment-934640347) on desktop Windows 1909.
> 
> I suspect this is the same root problem. Software archeology in progress (i.e. I'm digging :P) to verify Windows Server 2019 LTSC is based on 19H1 then yes, this should be addressed by #1681

Confirmed. Kudos to wikipedia for the convenient reference:

https://en.wikipedia.org/wiki/List_of_Microsoft_Windows_versions#Server_versions
<html>
<body>
<h2 style="color: rgb(0, 0, 0); margin: 1em 0px 0.25em; padding: 0px; overflow: hidden; border-bottom: 1px solid rgb(162, 169, 177); font-size: 1.5em; font-weight: normal; font-family: &quot;Linux Libertine&quot;, Georgia, Times, serif; line-height: 1.3; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><span class="mw-headline" id="Server_versions">Server versions</span><span class="mw-editsection" style="font-family: sans-serif; user-select: none; font-size: small; font-weight: normal; margin-left: 1em; vertical-align: baseline; line-height: 1em; unicode-bidi: isolate; margin-right: 0px;"><span class="mw-editsection-bracket" style="margin-right: 0.25em; color: rgb(84, 89, 93);">[</span><a href="https://en.wikipedia.org/w/index.php?title=List_of_Microsoft_Windows_versions&amp;action=edit&amp;section=3" title="Edit section: Server versions" style="text-decoration: none; color: rgb(6, 69, 173); background: none; white-space: nowrap;">edit</a><span class="mw-editsection-bracket" style="margin-left: 0.25em; color: rgb(84, 89, 93);">]</span></span></h2>

Name | Release date | Version number | Editions | Latest build | End of support
-- | -- | -- | -- | -- | --
Windows Server 2019 | 2018-11-13 | **1809** | Windows Server 2019<br>EssentialsWindows Server 2019<br>StandardWindows Server 2019 Datacenter | **17763** | 2024-01-09


</body>
</html>

Windows Server 2019 == 1809 == **RS5**

So yes, this should be addressed by #1681

**Comment by riverar** [comment]
@DrusTheAxe if you would like an extra set of eyes on this fix, ping me (and shoot over some build artifacts).
